sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_response_handlers.py

```diff
@@ -401,18 +401,25 @@ def _get_doc_results(task, doc_id_order, response_headers, returned_tasks_object
     if response_task_to_deserialize.results is None:
         return get_ordered_errors(returned_tasks_object, task_name, doc_id_order)
     return deserialization_callback(
-        doc_id_order, response_task_to_deserialize.results, response_headers, lro=True
+        doc_id_order, response_task_to_deserialize.results, {}, lro=True
```

Comment:
why are you cleaning out response headers

Reply:
it's empty and we don't use it -> https://github.com/Azure/azure-sdk-for-python/pull/24995#discussion_r907905511